src/controllers/controller.polarArea.js

```diff
@@ -56,7 +57,12 @@ export default class PolarAreaController extends DatasetController {
     const scale = this._cachedMeta.rScale;
     const centerX = scale.xCenter;
     const centerY = scale.yCenter;
-    const datasetStartAngle = scale.getIndexAngle(0) - 0.5 * PI;
+
+    if (isNumber(scale.getIndexAngle(0))) {
+      this._lastIndexAngle = scale.getIndexAngle(0);
```

Comment:
Not totally a fan of this name since it is only the lastIndexAngle if `getIndexAngle` returns `NaN` in all other cases it shoudl be `currentIndexAngle` but couldn't come to a better name

Reply:
Maybe it would be better to change the getIndexAngle to return the startAngle in case there are not labels?

Reply:
Yes it is, totally didn't think about that. Way less hacky solution 👍